spec/lexers/digdag_spec.rb

```diff
@@ -21,7 +21,7 @@
 
     it 'recognizes one line comment on last line even when not terminated by a new line (#360)' do
       assert_tokens_equal "+step1:\n  echo> Hello!\n",
-        ["Literal.String", "+step1"],
+        ["Name.Attribute", "+step1"],
```

Comment:
This does change the way Digdag is highlighted. But I think after the change it's 'more correct' than originally. In Digdag ["+" is a task](https://docs.digdag.io/workflow_definition.html#is-a-task) but since this is delegated to YAML lexer, I think this is OK. If Digdag wanted this highlighted differently it should do that itself.